api/src/test/groovy/com/okta/idx/sdk/api/client/IDXAuthenticationWrapperTest.groovy

```diff
@@ -348,6 +348,25 @@ class IDXAuthenticationWrapperTest {
         assertThat(authenticationResponse.getTokenResponse().getIdToken(), notNullValue())
     }
 
+    @Test
+    void beginTransactionWithActivationTokenTest() {
+
+        def requestExecutor = mock(RequestExecutor)
+        def idxClient = new BaseIDXClient(getClientConfiguration(), requestExecutor)
+        def idxAuthenticationWrapper = new IDXAuthenticationWrapper()
+        //replace idxClient with mock idxClient
+        setInternalState(idxAuthenticationWrapper, "client", idxClient)
+
+        setMockResponseWithBodyParamMatcher(requestExecutor, "interact", "activation_token", "interact-response", 200, MediaType.APPLICATION_JSON)
```

Comment:
Can we also test that it doesn't exist in the original use case?

Reply:
done